### ISSUE TITLE
Any plans to add support for high-frame rate timecode frame rates?

### ISSUE THREAD

**Issue by mbechard** [eoyilmaz/timecode#37]
Since that is starting to appear in some devices.
https://ieeexplore.ieee.org/document/7438725

**Comment by Eiken** [comment]
This is something we need as well

**Comment by eoyilmaz** [comment]
Yeah with that document that should be fun to implement. But I don't know when I can do it right now.

**Comment by Eiken** [comment]
We might take a stab at it and submit a MR

**Comment by eoyilmaz** [comment]
That would be incredibly nice of you, thank you

**Comment by Eiken** [comment]
@eoyilmaz so we have this working for our needs now (motion capture) but I am not 100% sure it follows the standard. Would it still be interesting with an MR? Then maybe someone familiar with the standards can code review it so that it is correct

**Comment by eoyilmaz** [comment]
Hey @Eiken yeah sure please create a MR

**Comment by eoyilmaz** [comment]
Thank you for the work btw

**Comment by eoyilmaz** [comment]
Is there any update on the PR
